examples/index.html

```diff
@@ -189,6 +189,26 @@ <h2 style="margin-top:20px">Save</h2>
 
   <footer>
     <hr style="margin:20px;"><center><a class="color:grey;" id="clear-cache">Clear offline cache</a></center>
+    <div class="row">
+      <div class="col-md-6">
+        <h2>Need Help?</h2>
+        <p>
+          Post a link to this and ask help from other community members on <a href="https://github.com/publiclab/image-sequencer/issues">PublicLab.org</a>
+        </p>
+        <p>
+          <a class="btn btn-primary" href="https://github.com/publiclab/image-sequencer/issues">Ask a question &raquo;</a>
+        </p>
+      </div>
+      <div class="col-md-6">
+        <h2>Improve this tool</h2>
+        <p>
+          This is an open source toolkit which you can help add to and improve on <a href="https://github.com/">Github</a>
+        </p>
+        <p>
+          <a class="btn btn-default" href="https://github.com/publiclab/image-sequencer">View the code</a>
```

Comment:
Also sine the `ask a question` is blue in colour can you please make `view the code` also in blue?

Reply:
Sure i will change it

Reply:
What abt the link in ask a question do i keep it same (github issues link)??

Reply:
Yes !